ocs_ci/ocs/resources/pod.py

```diff
@@ -1755,6 +1756,7 @@ def get_pod_logs(
     namespace (str): Namespace of the pod
     previous (bool): True, if pod previous log required. False otherwise.
     all_containers (bool): fetch logs from all containers of the resource
+    since (str): only return logs newer than a relative duration like 5s, 2m, or 3h.
```

Comment:
same as my above comment

Reply:
no, because we need to mention the s[second] m[minutes] and etc...